discovery_child_development/notebooks/labelling/prodigy/prodigy_requirements.txt

```diff
@@ -0,0 +1,5 @@
+prodigy==1.14.11
+python==3.10.13
```

Comment:
Should we instead change the dependencies in environment.yaml so that python is 3.10.13? Does it need to be 3.10.13, or is stipulating 3.10 enough?